website/public/js/controllers/groupsCtrl.js

```diff
@@ -16,6 +16,13 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
     }
 
     $scope.isMemberOfGroup = function(userid, group){
+
+      // If the group is a guild, just check for an intersection with the
+      // current user's guilds, rather than checking the members of the group.
+      if(group.type === 'guild') {
+        return _.detect(Groups.myGuilds(), function(g) { return g._id === group._id });
```

Comment:
...wait, if it wasn't returning "truthy"/"falsey" correctly, the ternary operator wouldn't work. So it should still operate just fine. It'd just not be returning strict boolean values.

FWIW, `!!_.detect...` will support greater minification.